tests/fixtures/issues/issue-865.php

```diff
@@ -6149,59 +6149,6 @@
         'lite' => false,
         'standard' => true,
     ],
-    'issue-865-DQ' => [
```

Comment:
I removed this test as it is now caught by the SeaMonkey generic file (invalid version number for SeaMonkey currently).